packages/web/src/pages/earn/pool/[pool-path]/add.tsx

```diff
@@ -38,19 +45,55 @@ export default function EarnAdd() {
     ];
   }, [data, width]);
 
+  const feeStr = useMemo(() => {
+    const feeTier = data?.fee;
+
+    if (!feeTier) {
+      return null;
+    }
+    return SwapFeeTierInfoMap[makeSwapFeeTier(feeTier)]?.rateStr;
+  }, [data?.fee]);
+
+  const seoInfo = useMemo(() => SEOInfo["/earn/pool/[pool-path]/add"], []);
+
+  const title = useMemo(() => {
+    const tokenAPath = data?.tokenA.path;
+    const tokenBPath = data?.tokenB.path;
+
+    const tokenA = getGnotPath(tokenAPath ? tokens.find(item => item.path === checkGnotPath(tokenAPath)) : undefined);
+    const tokenB = getGnotPath(tokenBPath ? tokens.find(item => item.path === checkGnotPath(tokenBPath)) : undefined);
+
+    return seoInfo.title([tokenA?.symbol, tokenB?.symbol, feeStr].filter(item => item) as string[]);
+  }, [
+    data?.tokenA.path,
+    data?.tokenB.path,
+    feeStr,
+    getGnotPath,
+    seoInfo,
+    tokens
+  ]);
+
   return (
-    <PoolAddLayout
-      header={<HeaderContainer />}
-      breadcrumbs={
-        <BreadcrumbsContainer
-          listBreadcrumb={listBreadcrumb}
-          isLoading={isLoadingCommon || isLoading}
-        />
-      }
-      addLiquidity={<PoolAddLiquidityContainer />}
-      oneStaking={<OneClickStakingContainer />}
-      exchangeRateGraph={<ExchangeRateGraphContainer />}
-      footer={<Footer />}
-    />
+    <>
+      <SEOHeader
+        title={title}
+        pageDescription={seoInfo.desc()}
+        ogTitle={seoInfo?.ogTitle?.()}
+        ogDescription={seoInfo?.ogDesc?.()}
+      />
```

Comment:
This is the code that goes into each page.
Isn't there a common way to handle it?

Reply:
I think they have same component with different data, but i'll check it after PR

Reply:
Okay, I'll leave this comment open.